.vscode/extensions.json

```diff
@@ -1,13 +1,14 @@
 {
   "recommendations": [
-    "nickytonline.dev-to-extension-pack",
-    "waderyan.nodejs-extension-pack",
-    "esbenp.prettier-vscode",
+    "42crunch.vscode-openapi",
     "donjayamanne.git-extension-pack",
-    "rebornix.ruby",
+    "esbenp.prettier-vscode",
+    "firefox-devtools.vscode-firefox-d",
     "ms-vsliveshare.vsliveshare-pack",
-    "42crunch.vscode-openapi",
     "philosowaffle.openapi-designer",
-    "silvenon.mdx"
+    "sbenp.prettier-vscode",
+    "silvenon.mdx",
+    "waderyan.nodejs-extension-pack",
+    "koichisasada.vscode-rdbg"
```

Comment:
I see this particular extension appears both here in the recommendeds list, and up in the devcontainer's auto-install list, is there a reason for this or are we able to consolidate?

Reply:
This is odd on VS Code's part: You can't force a user to install an extension, but you _can_ force a Dev container to install an extension. `extension.json` is the "recommendation" while `devcontainer.json` is a requirement.

Reply:
Much as I noted in the devcontainer file about commenting this in the source, can we go ahead and comment this in the source as well? That's really good context I would have never guessed, and [it appears this file is, like the others, parsed as JSONC](https://github.com/microsoft/vscode/blob/f92db2f96e8959d65dfc73bab7b2f5f21c2beefb/extensions/configuration-editing/package.json#L38-L57), allowing comments.

Reply:
Added!